tests/models/convbert/test_modeling_convbert.py

```diff
@@ -459,6 +459,11 @@ def test_model_for_input_embeds(self):
         result = model(inputs_embeds=inputs_embeds)
         self.assertEqual(result.last_hidden_state.shape, (batch_size, seq_length, config.hidden_size))
 
+    def test_reducing_attention_heads(self):
```

Comment:
this should not be included in the PR, please revert all changes unrelated

Reply:
It's been removed, this got mixed in when trying to rebase